testdata/baselines/reference/submodule/compiler/constEnums.js.diff

```diff
@@ -37,9 +37,9 @@
 +    // correct cases: reference to the enum member from different enum declaration
 +    Enum1["W1"] = A0;
 +    if (typeof Enum1.W1 !== "string") Enum1[Enum1.W1] = "W1";
-+    Enum1["W2"] = Enum1.A0;
++    Enum1["W2"] = 100 /* Enum1.A0 */;
```

Comment:
Not related to your PR, but this is an interesting case, since I assume this is due to us not doing enum merging properly or something